### ISSUE TITLE
Importing global PHP functions

### ISSUE THREAD

**Issue by szymach** [apache/netbeans#4527]
### Description

Hello!

Right now only functions from other namespaces can be imported (and `count` for some reason) with the `use function` directive. All other display the `cannot be resolved` in the `Fix imports in Current Namespace` window. 

### Use case/motivation

I have found the `use function` to be far more elegant than the prefixed syntax, but I have to do it manually. Being able to import them automatically like the class names would save a lot of tedious minor work.

### Are you willing to submit a pull request?

No

### Code of Conduct

Yes

**Comment by junichi11** [comment]
Could you also write an example code?

**Comment by szymach** [comment]
Does this explain my meaning, or should I also include how it should happen in the IDE?

```php

namespace Demo;

use function in_array;

class someClass 
{
    public function someMethod(string $key): bool
    {
        return in_array($key, ['a value'], true);
    }
}
```

**Comment by junichi11** [comment]
> should I also include how it should happen in the IDE

Yes, well, it would be nice if you write your exact steps, an actual result, and an expected result.

1. Copy & paste the example code
2. ...
3. ...

Actual result:

...

Expected result:

...

BTW, we are going to add the "Auto Import" feature in NetBeans 22 (https://github.com/apache/netbeans/pull/7027)
